funnel/views/project.py

```diff
@@ -460,6 +459,55 @@ def rsvp_transition(self):
     def rsvp_list(self):
         return {'project': self.obj, 'statuses': RSVP_STATUS}
 
+    def get_rsvp_state_csv(self, state):
+        outfile = io.StringIO()
+        out = csv.writer(outfile)
+        out.writerow(['fullname', 'email', 'created_at'])
+        for rsvp in self.obj.rsvps_with(state):
+            out.writerow(
+                [
+                    rsvp.user.fullname,
+                    rsvp.user.email,
+                    rsvp.created_at.astimezone(self.obj.timezone)
+                    .replace(second=0, microsecond=0, tzinfo=None)
+                    .isoformat(),  # Strip precision from timestamp
+                ]
+            )
+
+        outfile.seek(0)
+        return Response(
+            outfile.getvalue(),
+            content_type='text/csv',
+            headers=[
+                (
+                    'Content-Disposition',
+                    'attachment;filename="{filename}.csv"'.format(
+                        filename='rsvp-{project}-{state}'.format(
+                            project=make_name(self.obj.title), state=state
```

Comment:
`project=self.obj.name`

Reply:
can't do this because project names are mostly years. nor verbose.

Reply:
`profile-project`?

Reply:
That's what make_name on title does, right? Our do you want to avoid the function call?